desk/ted/posthog.hoon

```diff
@@ -0,0 +1,61 @@
+::  posthog: submit an log event to PostHog
+::
+/-  spider
+/+  io=strandio, l=logs
+::
+=+  posthog-key='phc_GyI5iD7kM6RRbb1hIU0fiGmTCh4ha44hthJYJ7a89td'
+=+  posthog-url='https://eu.i.posthog.com/capture/'
+=+  posthog-retry=3
+=+  posthog-retry-delay=~s30
```

Comment:
Could tune this down if you wanted, ~s5 is probably plenty if we're only doing three retries.